locations/kotti.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+location: kotti
+location_nice: Skalitzer Straße 134, 10999 Berlin
+latitude: 52.4992763
+longitude: 13.4195371
+altitude: 41
+height: 13
+community: true
+
+hosts:
+
+  - hostname: kotti-core
+    role: corerouter
+    model: "ubnt_unifiac-mesh"
+    wireless_profile: freifunk_default
+
+snmp_devices:
+  - hostname: kotti-simeon
+    address: 10.31.167.210
+    snmp_profile: airos_8
+
+ipv6_prefix: 2001:bf7:830:6600::/56
+
+# Kotti has following prefixes:
+# Router: 10.31.167.208/28
+# --MGMT: 10.31.167.208/29
+# --MESH: 10.31.167.216/29
+# --DHCP: 10.248.2.0/26
+
+networks:
```

Comment:
why don't we activate 802.11s mesh networks? I would activate them, so new neighbors could start meshing and get in contact.